src/vision/status_quo/frederik_makes_their_first_steps_into_async.md

```diff
@@ -0,0 +1,64 @@
+# 😱 Status quo stories: Frederik makes their first foray into async
+
+## Frederik's first big project in Rust: a journey marred by doubt
+
+It's Frederik's last year at their university and for their masters thesis, they have chosen to create a distributed database.
+They have chosen to use their favorite language, Rust, because Rust is a suitable language for low latency applications that they have found very pleasant to work in.
+Their project presents quite a challenge since they have only written some small algorithms in Rust, and it's also their first foray into creating a big distributed system.
+
+### Deciding to use Async
+Up until now, Frederik has followed the development of Async from afar by reading the occasional Boats blog post, and celebrating the release announcements with the rest of the happy community.
+Due to never having worked with async in other languages, and not having had a project suitable for async experimentation, their understanding of async and its ecosystem remained superficial.
+However, since they have heard that async is suitable for fast networked applications, they decide to try using async for their distributed database.
+After all, a fast networked application is exactly what they are trying to make.
+
+To further solidify the decision of using async, Frederik goes looking for some information and opinions on async in Rust. Doubts created by reading some tweets about how most people should be using threads instead of async for simplicity reasons are quickly washed away by helpful conversations on the Rust discord.
```

Comment:
I think the character should definitely be Barbara. She is an experienced Rust dev with ambiguous amounts of experience with other languages and other languages' async models. It's important to this story that the character knew a bit about the social norms of the Rust community for getting help making decisions (e.g., going to discord, knowing there might be a book for this topic, etc.).

Reply:
Thanks! I changed the character to Barbara and added some comments on how the experience might be different for others.

Reply:
Hmm, on second thought I read `Lately she's maintaining various projects in the async space.`

That seems to show already having significant experience with async, right? I feel like barbera's experience would be quite different since she would already know async.

Reply:
I think we can work around this by framing this as her first async project. There's going to be a lot of small tweaks we could make to all characters. I think it's best we try to make it work with the existing characters unless it's quite clear that no character fits.